demisto_sdk/commands/common/constants.py

```diff
@@ -479,22 +479,42 @@ def invalid_v2_script_name(file_path):
+PLAYBOOK_CHANGELOG_REGEX = r'{}(?!Test){}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, PLAYBOOKS_DIR)
+
 TEST_PLAYBOOK_REGEX = r'{}{}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
 TEST_NOT_PLAYBOOK_REGEX = r'{}{}/(?!playbook).*-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
 
 INCIDENT_TYPE_REGEX = r'{}{}/incidenttype-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_TYPES_DIR)
+INCIDENT_TYPE_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_TYPES_DIR)
+
 INDICATOR_FIELDS_REGEX = r'{}{}/incidentfield-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
+INDICATOR_FIELDS_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
+INCIDENT_FIELD_REGEX = r'{}{}/incidentfield-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_FIELDS_DIR)
+INCIDENT_FIELD_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_FIELDS_DIR)
+
 WIDGETS_REGEX = r'{}{}/widget-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, WIDGETS_DIR)
+WIDGETS_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, WIDGETS_DIR)
+
 DASHBOARD_REGEX = r'{}{}.*dashboard-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, DASHBOARDS_DIR)
+DASHBOARD_CHANGELOG_REGEX = r'{}{}.*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, DASHBOARD_REGEX)
+
 CONNECTIONS_REGEX = r'{}{}.*canvas-context-connections.*\.json$'.format(CAN_START_WITH_DOT_SLASH, CONNECTIONS_DIR)
+
 CLASSIFIER_REGEX = r'{}{}.*classifier-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, CLASSIFIERS_DIR)
+CLASSIFIER_CHANGELOG_REGEX = r'{}{}.*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, CLASSIFIERS_DIR)
+
 LAYOUT_REGEX = r'{}{}.*layout-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, LAYOUTS_DIR)
-INCIDENT_FIELD_REGEX = r'{}{}/incidentfield-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_FIELDS_DIR)
-MISC_REGEX = r'{}{}.*reputations\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
-REPUTATION_REGEX = r'{}{}/reputation-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
+LAYOUT_CHANGELOG_REGEX = r'{}{}.*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, LAYOUTS_DIR)
+
 REPORT_REGEX = r'{}{}.*report-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, REPORTS_DIR)
+REPORT_CHANGELOG_REGEX = r'{}{}.*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, REPORTS_DIR)
+
+MISC_REGEX = r'{}{}.*reputations\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
 MISC_REPUTATIONS_REGEX = r'{}{}.reputations.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
-README_REGEX = r'{}{}/readme.*\.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
+REPUTATION_REGEX = r'{}{}/reputation-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
+REPUTATION_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
+
+README_REGEX = r'{}{}/*_README.*\.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
```

Comment:
?

Reply:
line 517 is faulty(and not being used)
yana will add a proper readme for old JS integrations in: https://github.com/demisto/demisto-sdk/pull/275

Reply:
this is line 517, if it's faulty and not used - remove it